<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>peakDisplayDialog</class>
 <widget class="QDialog" name="peakDisplayDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>349</width>
    <height>164</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Peak Display Adjust</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>9</y>
     <width>331</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Image Select :</string>
   </property>
   <widget class="QRadioButton" name="rawRB">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Raw </string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="fittedRB">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Fitted Gaussian</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="residsRB">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>30</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Residual</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="updateAdjustButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>90</y>
     <width>100</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Update </string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>130</y>
     <width>100</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Minimum :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="minLE"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Maximum :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="maxLE"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
